src/components/WateringModal/index.tsx

```diff
@@ -18,12 +19,15 @@ const TwoColumnGrid = styled.div`
     grid-template-columns: repeat(2, 1fr);
   }
 `;
-
 const ButtonsContainer = styled.div`
   margin-top: 48px;
   display: flex;
   justify-content: space-between;
 `;
+const NotifictionContainer = styled(ButtonsContainer)`
+  justify-content: center;
+  width: 100%;
+`;
 
 const StyledError = styled.p`
```

Comment:
looks like the `const StyledError` is not in use anymore? can we delete it?

Reply:
True. Thank you